__mocks__/react-native-permissions.ts

```diff
@@ -1,12 +1,13 @@
 import * as RNPermission from 'react-native-permissions/lib/typescript';
```

Comment:
This is where the eslint is failing... there's an annotation.. 

```
'RNPermission' is defined but never used.
```

silly gh action didn't make this clear in its logs, though.

Reply:
awesome! great catch